### PR TITLE
Install loopback firewall rules for FTL

### PR DESCRIPTION
**Pull request by DL6ER**
**By submitting this pull request, I confirm the following (please check boxes, eg [X]) _Failure to fill the template will close your PR_:**

***Please submit all pull requests against the `development` branch. Failure to do so will delay or deny your request***

- [X] I have read and understood the [contributors guide](https://github.com/pi-hole/pi-hole/blob/master/CONTRIBUTING.md).
- [X] I have checked that [another pull request](https://github.com/pi-hole/pi-hole/pulls) for this purpose does not exist.
- [X] I have considered, and confirmed that this submission will be valuable to others.
- [X] I accept that this submission may not be used, and the pull request closed at the will of the maintainer.
- [X] I give this submission freely, and claim no ownership to its content.

**How familiar are you with the codebase?:** 10

---

Basically, what the title says. Still needs implementation of the rule for `firewallD`.

_This template was created based on the work of [`udemy-dl`](https://github.com/nishad/udemy-dl/blob/master/LICENSE)._

### CODE REVIEW

**Comment by DL6ER** [comment]
@dschaper Still need the command to set the same up for `FirewallD`.

**Comment by dschaper** [comment]
Add ports, but still need to tie it to localhost only.

**Comment by dschaper** [comment]
Would adding localhost to the trusted zone and then adding the ports to the trusted zone be acceptable? @bcambl

**Comment by bcambl** [comment]
Are you experiencing localhost traffic being blocked by firewalld?

**Comment by dschaper** [comment]
Not personally, but dl6er thinks that may be something that is an issue.

**Comment by bcambl** [comment]
You could maybe create a new `FTL` zone, add `lo` to it, then add the ports.
@DL6ER, do you have an environment where you are experiencing an issue with firewalld blocking local?
Leaving the firewall up and [listening local](https://github.com/pi-hole/FTL/blob/master/config.c#L35)  _should_  be fine.

**Comment by DL6ER** [comment]
@bcambl No, I haven't seen this with `firewallD`.

However, we have seen several reports connected to `iptables`. I thought that a consistent behavior would be a good idea, so I requested the changes for `firewallD` to be done by someone who knows how to do that.

**Comment by dschaper** [comment]
Rather than mess with adding zones I think just `iptables` would be good. I would say we can proceed without the modifications to FirewallD.
